rakelib/plugin.rake

```diff
@@ -23,7 +23,9 @@ namespace "plugin" do
 
   task "install-all" => [ "dependency:octokit" ] do
     Rake::Task["vendor:bundle"].invoke("tools/Gemfile.plugins.all")
+    if ENV["LS_ENV"] == "test"
```

Comment:
We can do, my first idea as to have it all in one place as I use to find it easier.